src/handler/src/commands/github/issue-pr.ts

```diff
@@ -0,0 +1,447 @@
+			const groups = imageRegex.exec(issue.body)?.groups;
+
+			// footer icon
+			const icon_url = isPR(issue)
+				? PRIcons[resultState as ResultStatePR]
+				: IssueIcons[resultState as ResultStateIssue];
+
+			// footer text
+			const comments = issue.comments.totalCount
+				? `(${IssuePRLookup.formatCommentString(issue.comments.totalCount, locale)})`
+				: '';
+
+			const isMerge = isPR(issue) && resultState === 'MERGED';
+			const user = isPR(issue) && resultState === 'MERGED' ? issue.mergedBy?.login : undefined;
+			const commit = isPR(issue) && resultState === 'MERGED' ? issue.mergeCommit?.abbreviatedOid : undefined;
+
+			const action = isMerge
+				? user && commit
+					? i18next.t('command.issue-pr.execute.action.merge_by_in', { lng: locale, user, commit })
+					: user
+					? i18next.t('command.issue-pr.execute.action.merge_by', { lng: locale, user })
+					: commit
+					? i18next.t('command.issue-pr.execute.action.merge_in', { lng: locale, commit })
+					: i18next.t('command.issue-pr.execute.action.merge', { lng: locale })
+				: resultState === 'CLOSED'
+				? i18next.t('command.issue-pr.execute.action.close', { lng: locale })
+				: resultState === 'DRAFT'
+				? i18next.t('command.issue-pr.execute.action.draft', { lng: locale })
+				: i18next.t('command.issue-pr.execute.action.open', { lng: locale });
+
+			const text = `${comments}${action}`;
+
+			// timestamp
+			const timestampProperty = Timestamps[resultState];
+
+			const e1: Embed = {
+				author: {
+					icon_url: `${issue.author.avatarUrl}?anticache=${Date.now()}`,
+					name: issue.author.login,
+					url: issue.author.url,
+				},
+				title: `#${issue.number} ${issue.title}`,
+				url: issue.url,
+				footer: { text, icon_url },
+				description: verbose ? IssuePRLookup.formatBody(issue.body) : undefined,
+				color: StateColors[resultState],
+				image: verbose && groups ? { url: groups.url } : undefined,
+				timestamp: isPR(issue) ? issue[timestampProperty]! : issue[timestampProperty as TimestampsWithoutMergedKey]!,
+			};
+
+			// install with
+			const installable = resultState in InstallableState;
+			const e2: Embed =
+				isPR(issue) && installable
+					? addField(e1, {
+							name: i18next.t('command.issue-pr.execute.headings.install', { lng: locale }),
+							value: `\`npm i ${issue.headRepository.nameWithOwner}#${
+								issue.headRef?.name ?? i18next.t('command.issue-pr.execute.unknown', { lng: locale }) ?? ''
+							}\``,
+					  })
+					: e1;
+
+			// reviews
+			const reviews = isPR(issue) ? IssuePRLookup.relevantReviews(issue.author.login, issue.reviews.nodes) : [];
+			const reviewBody = reviews
+				.map((r) => {
+					const isDjsMember =
+						owner === 'discordjs' && ['MEMBER', 'OWNER', 'COLLABORATOR'].includes(r.authorAssociation);
+					const reviewBadge = isDjsMember ? Badges.DJS : '';
+					const reviewLink = `[${r.author.login}](${r.url})`;
+					const reviewState = `${IssuePRLookup.cleanDecision(r.state)}`;
```

Comment:
```suggestion
					const reviewState = IssuePRLookup.cleanDecision(r.state);
```
function returns a string